src/test/scala/uk/gov/hmrc/test/ui/pages/common/DashboardPage.scala

```diff
@@ -32,15 +32,14 @@ object DashboardPage extends BasePage {
 
   override def checkExpanders(): Unit = ()
 
-  private val messageLink = "/messages"
-  override def pageLinkHrefs: Seq[String] = super.pageLinkHrefs.filterNot(_ == exitAndCompleteLater) :+ messageLink
+  override def pageLinkHrefs: Seq[String] = super.pageLinkHrefs.filterNot(_ == exitAndCompleteLater)
 
   override def fillPage(values: String*): Unit = ()
 
   // validateDashboard("Submitted", "Declaration submitted")
```

Comment:
Remove this

Reply:
This is a comment to show what needs passing in validateDashboard method. So no need to remove this